src/renderer/entities/governance/model/governance-model.ts

```diff
@@ -0,0 +1,29 @@
+import { createStore } from 'effector';
+
+import type {
+  TrackId,
+  TrackInfo,
+  ReferendumId,
+  OngoingReferendum,
+  CompletedReferendum,
+  VotingThreshold,
+  Voting,
+  Address,
+} from '@shared/core';
+
+const $ongoingReferendums = createStore<Map<ReferendumId, OngoingReferendum>>(new Map());
```

Comment:
Is it okay to set store directly from another store? I thought that we should use only events inside other models

Reply:
I've introduced `entities/governance` to hold all the data required for governance features/screens
So every feature that requests data can safely set entity's store, and we already do that in some places `rename-wallet/vault-details/wallet-connect`